.travis.yml

```diff
@@ -6,17 +6,20 @@ services:
 language: go
 go:
  - 1.9
+ - tip
 
 go_import_path: github.com/ONSdigital/rm-survey-service
 
 script:
   - make build test docker
+  - go test -race -coverprofile=coverage.txt -covermode=atomic
```

Comment:
Maybe try adding `-race -coverprofile=coverage.txt -covermode=atomic` to the test section in the make file?